packages/create/templates/dbos-knex/knexfile.js

```diff
@@ -1,7 +1,6 @@
-const { readConfigFile, getDatabaseUrl } = require('@dbos-inc/dbos-sdk');
-
-const dbosConfig = readConfigFile();
-const databaseUrl = getDatabaseUrl(dbosConfig);
+const databaseUrl =
```

Comment:
We should use the setup-db script from the demo apps instead of knex file. Knexfile does not create the database.